src/test/java/app/persistence/OrderMapperTest.java

```diff
@@ -0,0 +1,210 @@
+package app.persistence;
+
+import app.exceptions.DatabaseException;
+import app.exceptions.OrderCreationException;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class OrderMapperTest {
+    private static final String USER = "postgres";
+    private static final String PASSWORD = "postgres";
+    private static final String URL = "jdbc:postgresql://localhost:5432/%s?currentSchema=test";
+    private static final String DB = "fog";
+
+    private static final ConnectionPool connectionPool = ConnectionPool.getInstance(USER, PASSWORD, URL, DB);
+
+    @BeforeAll
+    public static void setUpClass() {
```

Comment:
Er det muligt at kalde AccountMapperTest.setUpClass() så vi ikke behøver at have det samme kode stående to steder?
(Jeg ved det ikke selv. Jeg fik kigget lidt på opsætning af integrationstest i efterårsferien og kom frem til at det er sværere end jeg troede)